components/pages/submission-system/program-submitted-data/SearchBar.tsx

```diff
@@ -0,0 +1,150 @@
+import DropdownButton from '@icgc-argo/uikit/DropdownButton';
+import { useTheme } from '@icgc-argo/uikit/ThemeProvider';
+import {
+  searchBackgroundStyle,
+  searchTitleParentStyle,
+  searchBoldTextStyle,
+  searchClearFilterStyle,
+  searchRightSideGroupStyle,
+  searchFilterParentStyle,
+  searchDropdownStyle,
+  searchDownArrowStyle,
+  searchBarParentStyle,
+  searchInputFieldStyle,
+  searchFilterButtonStyle,
+  searchFilterIconStyle,
+  searchDownloadIconStyle,
+} from './style';
+import Icon from '@icgc-argo/uikit/Icon';
+import { Input } from '@icgc-argo/uikit/form';
+import Button from '@icgc-argo/uikit/Button';
+
+import { css } from '@icgc-argo/uikit';
+import { DownloadIcon } from 'components/pages/file-entity/common';
+
+export default function SearchBar({ noData }: { noData: boolean }) {
+  const theme = useTheme();
+
+  const [keyword, setKeyword] = useState('');
+  useEffect(() => {}, []);
+
+  return (
+    <Container css={searchBackgroundStyle}>
+      {/* First Item - title */}
+      <div css={searchTitleParentStyle}>
+        Clincial Data for:{' '}
+        <b css={searchBoldTextStyle}>{keyword ? ` ${keyword}` : ` All Donors`}</b>
+        {keyword && (
+          <Button
+            onClick={() => {
+              setKeyword('');
+            }}
+            css={searchClearFilterStyle}
+            variant="secondary"
+          >
+            <u>Clear Filters</u>
+          </Button>
+        )}
+      </div>
+
+      {/* Grouping second to forth item together */}
+      <div css={searchRightSideGroupStyle}>
+        {/* Second item - filter */}
+        <div css={searchFilterParentStyle}>
+          Quick Filters:
+          <DropdownButton
+            css={searchDropdownStyle}
+            variant="secondary"
+            size="sm"
+            onItemClick={() => {}}
+            menuItems={[
+              {
+                display: 'Show all donors',
+                value: 'all',
+              },
+              {
+                display: 'Show invalid donors',
+                value: 'all',
+              },
+              {
+                display: 'Show clinically complete donors',
+                value: 'all',
+              },
+              {
+                display: 'Show clinically incomplete donors',
+                value: 'all',
+              },
+            ]}
+          >
+            - select an option -
+            <Icon name="chevron_down" fill="accent2_dark" css={searchDownArrowStyle} />
+          </DropdownButton>
+        </div>
+
+        {/* Third item - search bar */}
+        <div css={searchBarParentStyle}>
+          <Input
+            aria-label="search-for-files"
+            size="sm"
+            placeholder={'Donor ID/Submitter Donor ID'}
+            preset="search"
+            showClear={true}
+            value={keyword}
+            onChange={(e) => {
+              setKeyword(e.target.value);
+            }}
+            getOverrideCss={() => searchInputFieldStyle}
```

Comment:
Remove gap in hover state:

```suggestion
            getOverrideCss={() => css`
              ${searchInputFieldStyle}
              margin-right: 0;
            `}
```

Reply:
resolved this by editing css in variable, `searchInputFieldStyle` during our peer review.